Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/exchangemgr/ExchangeScheduledTask.java

```diff
@@ -62,18 +62,21 @@
     private static final Logger LOG = LoggerFactory.getLogger(ExchangeScheduledTask.class);
     private static final ArrayList<String> backupFileList = new ArrayList<>();
     private static final String JSON_QUERY_PARAM = "_format=json";
+    private static boolean hasDownloadOccurred = false;
 
     public void task() {
         try {
             LOG.info("Starting ExchangeSceduleTask");
             ExchangeInfoType exInfo = getExchangeDAO().loadExchangeInfo();
             if (null != exInfo) {
                 for (ExchangeType ex : exInfo.getExchanges().getExchange()) {
-                    fetchExchangeData(ex);
+                    hasDownloadOccurred = hasDownloadOccurred || fetchExchangeData(ex);
+                }
+                if (hasDownloadOccurred) {
```

Comment:
does this variable always true regardless how many time this method run?

Reply:
This variable will evaluate to false if download on all the exchanges listed in exchangeInfo.xml is disabled